workshop_operations/checklists.md

```diff
@@ -79,44 +75,45 @@ Table of Contents
 * if coffee, tea, or water runs out contact Carpentries@UiO-coordinator or the library contact and see if more can be arranged
 * help collecting workshop equipment after the workshop
 * help setting the room
+* be attentive: you likely notice participants' challenges and problems much better than the instructors
+* be kind to yourselves: no one expects you to know solutions to all the problems
 
 # University Library
 
 ## Initial Preparation
 
-* book rooms for the semester and create schedule in spreadsheet
-* set up a registration form (nettskjema) as well as a waiting list and give lead instructor (and co-instructors and/or helpers) access to it
-* ask lead instructor for the URL of the workshop information page (usually uio-carpentry.github.uio/YYYY-DD-MM-topic)
-* set up an event on uio.no website
+* book rooms for the semester
+* send out calls for co-instructors and helpers
+* set up a registration form (Pindena)
+* ask lead instructor for the URL of the workshop information page (usually uio-carpentry.github.uio/YYYY-DD-MM-uio)
+* set up an event on uio.no course catalogue
+* advertise event by sending an email to ``announcements @ swcarpentry dot uio dot no`` and other relevant lists and via the official social media channels of the University Library
 
-## Several Weeks Before
+## Two Weeks Before
 
-* share the pre- & post-workshop questionnaire with the lead instructor
+* send out a welcome message to the participants reminding them of installation requirements etc.
 
 ## One week before
 
-* order coffee/tea
+* order coffee/tea & fruit
 * ask lead instructor for equipment requirements (e.g., second monitor + laptop, microphone)
 * arrange for workshop room visit with (lead) instructor(s) and optionally helpers
 * arrange for a workshop-contact person for the day(s) of the workshop and give contact information to lead instructor
 
 ## At/during the workshop
 
-* prepare signs directing learners to the workshop room, these can be found in this repository, [here]()
-* take the coffee and tea to the workshop room
+* prepare signs directing learners to the workshop room, these can be found in this repositorys
```

Comment:
```suggestion
* prepare signs directing learners to the workshop room, these can be found in this repository
```

Reply:
I see – problem is: there are no templates for signs in this repository! 
I hope we can get a roll-up poster for Carpentry@UiO which will be visible and serve as an advertisement for the community, too.
Until this is done, I would like to remove the line above until we have a solution.
Ok?